bundles/io/org.openhab.io.caldav/.classpath

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" output="target/classes" path="src/main/java"/>
+	<classpathentry excluding="**" kind="src" output="target/classes" path="src/main/resources"/>
+	<classpathentry exported="true" kind="lib" path="lib/sardine-5.3.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/commons-lang3-3.3.2.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/httpclient-4.3.5.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/httpcore-4.3.2.jar"/>
```

Comment:
do you really need httpclient 4.3.5 (assuming it is the Apache HttpClient)? 4.1 is available in the existing target platform